platform/jobs/RetirementJobEdxTriggers.groovy

```diff
@@ -57,22 +57,6 @@ List jobConfigs = [
         extraMembersCanBuild: [],
         cron: '30 7 * * *',  // 08:30 UTC every day.
         disabled: true
-    ],
-    [
-        downstreamJobName: 'user-retirement-archiver',
-        triggerJobNamePrefix: 'user-retirement-archiver-trigger',
-        environmentDeployment: 'prod-edx',
-        extraMembersCanBuild: [],
-        cron: '0 9 * * *',  // 10:00 UTC every day.
-        disabled: true
-    ],
-    [
-        downstreamJobName: 'user-retirement-archiver',
-        triggerJobNamePrefix: 'user-retirement-archiver-trigger',
-        environmentDeployment: 'prod-edge',
-        extraMembersCanBuild: [],
-        cron: '30 9 * * *',  // 10:30 UTC every day.
-        disabled: true
```

Comment:
After merging this, I'll try to remember to go back and delete these old trigger jobs.